### PR TITLE
Update to image 4139

### PR DESCRIPTION
**Pull request by bormanp**

### CODE REVIEW

File: manifests/keysight/ixiatg-configmap-unlicensed.yaml [hunk 1]
```diff
@@ -11,12 +11,12 @@ data:
             {
                 "name": "controller",
                 "path": "ghcr.io/open-traffic-generator/ixia-c-controller",
-                "tag": "0.0.1-4435"
+                "tag": "0.0.1-4139"
```
Comment (alexmasi) [inline manifests/keysight/ixiatg-configmap-unlicensed.yaml:14]: this is a downgrade?

Comment (bormanp) [reply manifests/keysight/ixiatg-configmap-unlicensed.yaml:14]: Sigh.  Thank you git.
Had issues with branches.  Thanks for catching this.

**Comment by bormanp** [comment]
This has the right numbers now.

**Comment by alexmasi** [comment]
https://github.com/openconfig/kne/pull/428 does what we need here

**Pull request closed without merging**